src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/RandomStressTest/Randomizer.cs

```diff
@@ -237,18 +237,21 @@ protected static int DeserializeInt(byte[] buf, ref int offset)
             return unchecked((int)uval);
         }
 
+        //this method is only used for Randomizer constructor to avoid using overridable method inside constructor.
+        private void DeserializeInitializer()
+        {
+            string base64State = GetCurrentState().ToString();
+            Deserialize(Convert.FromBase64String(base64State), out _);
+        }
 
         /// <summary>
         /// deserialization constructor
         /// </summary>
         public Randomizer(StreamingContext context)
         {
-            string base64State = GetCurrentState().ToString();
-            int offset;
-            Deserialize(Convert.FromBase64String(base64State), out offset);
+            DeserializeInitializer();
```

Comment:
This constructor has no reference, can be removed instead of delegating virtual method call.

Reply:
done.